perfmetrics/scripts/bigquery/experiments_gcsfuse_bq.py

```diff
@@ -212,6 +212,7 @@ def setup_dataset_and_tables(self):
       CREATE TABLE IF NOT EXISTS {}.{}.{}(
         configuration_id STRING,
         start_time_build INT64,
+        mount_type STRING,
```

Comment:
For FYI @raj-prince 
This is for pd MOUNT_TYPE_GCS = 'gcs_bucket' MOUNT_TYPE_PD = 'persistent_disk'

Reply:
Do we use this field in the current setup? If not then we should remove this.

Reply:
Yes, we are